Modules/CLI/ExecutionModelTour/ExecutionModelTour.xml

```diff
@@ -190,8 +190,22 @@
     <description><![CDATA[Parameters that describe point sets.]]></description>
     <point multiple="true" coordinateSystem="ras">
       <name>seed</name>
-      <label>Seeds</label>
-      <longflag>--seed</longflag>
```

Comment:
Should we leave in --seed as a synonym for --seed_ras for backward compatibility?